packages/fe-mockserver-core/src/router/serviceRouter.ts

```diff
@@ -62,7 +63,11 @@ export async function serviceRouter(service: ServiceConfigEx, dataAccess: DataAc
         const parser = raw({ type: '*/*' });
         const tenantId = req.originalUrl?.startsWith('/tenant-') ? req.originalUrl?.split('/')[1] : 'tenant-default';
         req.tenantId = tenantId;
-
+        const sapClient = req.originalUrl?.includes('sap-client');
+        if (sapClient) {
+            const parsedUrl = new URL(`http://dummy${req.originalUrl}`);
```

Comment:
`new URL(req.originalUrl, 'http://dummy');` would work, too

Reply:
yes but i'm keeping it inline with the other code that does it the same way as here. might change later ;)